roles/os_env_extras_node/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- copy: src=enter-container.sh dest=/usr/local/bin/enter-container.sh mode=0755
```

Comment:
You are correct.  I kept it since there was some error generated if dockerroot was not found.